docs/supervisor-notes.md

```diff
@@ -1,14 +1,21 @@
-# vSphere with Kubernetes notes
+# vSphere with Tanzu notes
+
+## Networking Considerations
+
+**vSphere with Tanzu** supports two distinct networking deployments with the release of vSphere 7.0 Update 1 (U1). The first deployment leverages ```NSX-T``` to provide load balancing services as well as overlay networking for Pod VM to Pod VM communication in the Supervisor Cluster. The second networking configuration supported by vSphere with Tanzu uses native vSphere network distributed switches and a ```HA Proxy``` to provide load balancing services. Native vSphere networking does not provide any overlay networking capabilities, and thus this deployment does not currently support Pod VMs in the Supervisor. Since Pod VMs are a requirement when wish to use the **Velero Plugin for vSphere** in vSphere with Tanzu, it currently requires a networking deployment that uses NSX-T. vSphere with Tanzu deployments that use native vSphere distributed switch networking and the HA Proxy for load balancing cannot currently be backed up using the Velero Plugin for vSphere.
```

Comment:
These two paragraphs in this section are the same as "Important note on vSphere with Tanzu networking requirements" in readme.md.  Do we need them in both places?

Reply:
I think it is important enough to leave in both places. I added a line at the beginning to say that this is a repeat of what was in the README.md.